### PR TITLE
Fix description markup to link to README.md

### PR DESCRIPTION
**Pull request by mossprescott**
Closed the other PR and tried again and this time it defaults to targeting upstream/master.

### CODE REVIEW

**Comment by dfithian** [comment]
The link doesn't work :(

File: package.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ license: MIT
 synopsis: 'Runtime interpolation of environment variables in records using profunctors'
 description: |
   Runtime interpolation of environment variables in records using profunctors. See
-  `https://github.com/tvision-insights/interpolator/README.md`.
+  the [README](https://github.com/tvision-insights/interpolator/README.md).
```
Comment (dfithian) [inline package.yaml]: I think we can just link to the github repo and not the readme

**Comment by DanBurton** [comment]
n.b. you could also link to https://github.com/tvision-insights/interpolator#readme

**Comment by mossprescott** [comment]
Oh, that's better. I didn't know about that. I need a browser plugin that shows me all the anchors on the page, or something.

**Pull request merged**